src/html.htl

```diff
@@ -1,9 +1,23 @@
 <!DOCTYPE html>
 <html>
 <head>
-    <title>${content.title}</title>
-    <meta data-sly-test.hash=${content.data.sourceHash} name="x-source-hash" content="${hash}">
-    <esi:include src="/head.html" onerror="continue"/>
+  <title>${content.title}</title>
+  <meta data-sly-test.hash=${content.data.sourceHash} name="x-source-hash" content="${hash}"/>
+  <link data-sly-test="${content.meta.url}" rel="canonical" href="${content.meta.url}"/>
+  <meta data-sly-test.desc="${content.meta.description}" name="description" content="${content.meta.description}"/>
```

Comment:
You should be able to reuse the `desc` identifier here and on line 9.

Reply:
Good catch! Actually a leftover which I should have removed; I was playing with reusable sly variables at first, but then decided against it, because we stay more flexible that way in terms of moving things around, and using sly variables you have to pay attention to the order of the tags... I found the small character savings in the value attribute weren't worth it.